### PR TITLE
9479 document title/description, cleanup, redirects

### PR DESCRIPTION
**Pull request by rosiehsmith**
This PR includes:
- populating the description of the stamp order on "Add Docket Entry"
- cleanup `isMotion` as it's now being handled in a specific interactor
- add missing unit tests
- redirect to previously selected document if back is clicked off Apply Stamp form
- upon saving the stamp, go to the most recently created draft
- fix alignment of the back button

Generated document title
image
image

### CODE REVIEW

**Review by TomElliottFlexion — APPROVED** [review]

File: shared/src/proxies/documents/addDraftStampOrderDocketEntryProxy.js [hunk 1]
```diff
@@ -13,11 +13,18 @@ const { post } = require('../requests');
  */
 exports.addDraftStampOrderDocketEntryInteractor = (
   applicationContext,
-  { docketNumber, originalDocketEntryId, signedDocketEntryId, stampData },
+  {
+    docketNumber,
+    formattedDraftDocumentTitle,
```
Comment (jamesobrooks) [inline shared/src/proxies/documents/addDraftStampOrderDocketEntryProxy.js:19]: This one should be added to the JSDocs.

File: shared/src/business/useCases/addDraftStampOrderDocketEntryInteractor.js [hunk 2]
```diff
@@ -17,7 +17,13 @@ const { DocketEntry } = require('../entities/DocketEntry');
  */
 exports.addDraftStampOrderDocketEntryInteractor = async (
   applicationContext,
-  { docketNumber, originalDocketEntryId, signedDocketEntryId, stampData },
+  {
+    docketNumber,
+    formattedDraftDocumentTitle,
```
Comment (jamesobrooks) [inline shared/src/business/useCases/addDraftStampOrderDocketEntryInteractor.js:23]: This one should be added to the JSDocs.

File: web-client/src/styles/custom.scss [hunk 3]
```diff
@@ -1681,7 +1681,7 @@ button.change-scanner-button {
 .docket-entry-preview-text {
   overflow: hidden;
   width: 600px;
-  max-height: 41px;
+  max-height: 70px;
```
Comment (jamesobrooks) [inline web-client/src/styles/custom.scss:1684]: 🌱 I wonder if we should look deeper into this preview with UX, in the future.

File: web-client/src/views/Messages/MessageDocument.jsx [hunk 4]
```diff
@@ -165,7 +167,15 @@ export const MessageDocument = connect(
         )}
 
         {messageDocumentHelper.showApplyStampButton && (
-          <Button link href={messageDocumentHelper.applyStampLink} icon="stamp">
+          <Button
+            link
+            icon="stamp"
+            onClick={() => {
+              navigateToPathSequence({
```
Comment (jamesobrooks) [inline web-client/src/views/Messages/MessageDocument.jsx:174]: 🤔 I wonder if we should use the `navigateToPathAndSetRedirectUrlSequence` like we do in `DocumentViewerDocument.jsx` to ensure the behavior of the back button doesn't get broken. Or maybe we should have an integration test that verifies the behavior of the back button is correct?

**Review by jamesobrooks — CHANGES_REQUESTED** [review]
We should get the JSDocs updated before merging. Otherwise looks good!

File: web-client/src/styles/custom.scss (hunk 3 above)
Comment (rosiehsmith) [reply web-client/src/styles/custom.scss:1684]: noted that in the https://github.com/ustaxcourt/ef-cms/pull/2385!

File: web-client/src/views/Messages/MessageDocument.jsx (hunk 4 above)
Comment (rosiehsmith) [reply web-client/src/views/Messages/MessageDocument.jsx:174]: noted in the PR to test! https://github.com/ustaxcourt/ef-cms/pull/2385

**Review by jamesobrooks — APPROVED** [review]

**Pull request merged**